play-validations/memory-footprint/src/main/java/com/google/wear/watchface/dfx/memory/EvaluationSettings.java

```diff
@@ -38,7 +38,7 @@ final class EvaluationSettings {
 
     private static final String HONEYFACE_VERSION = "honeyface";
 
-    private static final List<String> SUPPORTED_VERSIONS = ImmutableList.of(HONEYFACE_VERSION, "1");
+    private static final List<String> SUPPORTED_VERSIONS = ImmutableList.of(HONEYFACE_VERSION, "1", "2");
```

Comment:
Probably worth a comment that this much the versions supported by WatchFaceXmlValidator?